gpkit/constraints/sigeq.py

```diff
@@ -0,0 +1,15 @@
+"Implements SignomialEquality"
+from .set import ConstraintSet
+from ..nomials import SingleSignomialEquality
+from ..nomials.array import array_constraint
+
```

Comment:
ugh, this class seems awkward. Are we going to have to define wrappers like this every time we write our own constraint class? It seems to me that array_constraint ought to handle this wrapping?